### PR TITLE
[14.0][MIG] sale_order_type

### PR DESCRIPTION
**Pull request by mohamedhagag**

### CODE REVIEW

File: sale_order_type/views/account_move_views.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<odoo>
+    <record id="sot_invoice_form" model="ir.ui.view">
+        <field name="name">account.move.form.sot</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <field name="journal_id" position="before">
```
Comment (newtratip) [inline sale_order_type/views/account_move_views.xml]:
```suggestion
            <label for="journal_id" position="before">
```

File: sale_order_type/views/account_move_views.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+                <field name="move_type" invisible="1" />
+                <field
+                    name="sale_type_id"
+                    attrs="{'invisible': [('move_type', 'not in', ['out_invoice', 'out_refund'])]}"
+                />
+            </field>
```
Comment (newtratip) [inline sale_order_type/views/account_move_views.xml]:
```suggestion
            </label>
```

**Comment by newtratip** [comment]
@mohamedhagag please change the title to `[14.0][MIG] sale_order_type`

**Comment by newtratip** [comment]
@mohamedhagag CI is red, please check it.

**Comment by newtratip** [comment]
@mohamedhagag, Do you have a plan to continue this ?

**Comment by pedrobaeza** [comment]
This was already migrated, sorry.

**Pull request closed without merging**